dragonfly/implementations/psyllid_provider.py

```diff
@@ -37,9 +37,10 @@ def __init__(self,
         self.channel_dict = {'a': 'ch0', 'b': 'ch1', 'c': 'ch2'}
         self.freq_dict = {'a': None, 'b': None, 'c': None}
         self.mode_dict = {'a':None, 'b':None, 'c':None}
+        self.mode_testing = False
 
 
-    def _finish_configure(self):
+    def check_all_psyllid_instances(self):
```

Comment:
We can remove this method if you like. It used to be the setup call method but we decided on the call to not do setup calls anymore. The operator could use it if he wanted to check how many psyllid instances are running and which instances are in triggering or in streaming mode.
But he can also ask the r2_channel_x_interfaces whether the corresponding psyllid instance is in triggering or streaming mode. So it is actually redundant.
What do you say? Should we just remove it?

Reply:
If it works then I don't think we can just leave it. The daq interface is here to mostly prevent "a user" from needing to go talk to the r2_* objects directly for "normal" use. If at some point the method is broken then we can consider removing it rather than maintaining it.

The docstring comment is a valid one, but is pervasive. It would be useful to add at least a short docstring for each function. I think we can merge this PR, add an open issue to add those, and let Christine add/merge those directly. That way the code changes can be deployed whenever we're ready to re-deploy.